ast/ast_test.go

```diff
@@ -549,6 +549,34 @@ broccoli = green
 	require.Equal(t, expected, string(convertASTToBytes(file)))
 }
 
+func TestAddCommentsAfterToSection(t *testing.T) {
+	config := `[fruits]
+fruit = apple ;; a comment
+fruit = papaya
+
+[food "vegetables"]
+broccoli = green
+broccoli = red
+`
+	file := Read(strings.NewReader(config))
+	file, ok := AddCommentsAfterToSection(file, []string{"; comment1"}, "fruits", "")
```

Comment:
This seems a little specific. Can't we just get the section:

```
file.Section("fruits", "").Field[0].CommentAfter = []string{"; comment1}
```

Reply:
This doesn't work, I don't think, because you can't access the section by name? So you need the function to iterate through the Sections slice, right?